src/app/operators/dialogs/box-plot-operator/box-plot-operator.component.ts

```diff
@@ -10,33 +10,38 @@ import {DataType, DataTypes} from '../../datatype.model';
 import {NumericPipe} from '../scatter-plot-operator/scatter-plot-operator.pipe';
 import {WaveValidators} from '../../../util/form.validators';
 import {BoxPlotType} from '../../types/boxplot-type.model';
+import {BehaviorSubject} from 'rxjs';
 
 @Component({
     selector: 'wave-box-plot-operator',
     templateUrl: './box-plot-operator.component.html',
     styleUrls: ['./box-plot-operator.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class BoxPlotComponent implements OnInit, AfterViewInit {
+export class BoxPlotComponent implements ControlValueAccessor, OnInit, AfterViewInit {
 
     form: FormGroup;
 
     pointLayers: Array<Layer<Symbology>>;
```

Comment:
Just a naming issue: I think it is obsolete that the layer is solely for points.